src/core/agent/services/ipexCommunicationService.ts

```diff
@@ -466,7 +466,7 @@ class IpexCommunicationService extends AgentService {
 
   async getAcdcFromIpexGrant(
     said: string
-  ): Promise<Omit<ACDCDetails, "status" | "credentialType" | "issuanceDate">> {
+  ): Promise<Omit<ACDCDetails, "status">> {
```

Comment:
Maybe we can just return `status: pending` here so we don't need to do it in the UI? Guess would be cleaner